README.md

```diff
@@ -3,3 +3,19 @@
 An experimental attempt at using [LDK](https://github.com/lightningdevkit/rust-lightning) to implement [bolt 12](https://github.com/lightning/bolts/pull/798) features for [LND](https://github.com/lightningnetwork/lnd).
 
 So, basically, Frankeinstein's monster. With extra steps.
+
+### Setting up LNDK
+
+To run `LNDK`, `LND` is assumed to be running. For directions on how to do this, try [this guide](https://docs.lightning.engineering/lightning-network-tools/lnd/run-lnd).
+
+When compiling `LND`, make sure that the peersrpc and signerrpc services are enabled, like this:
+
+`make install --tags="peersrpc and signerrpc`
+
+In order to successfully connect to `LND`, we need to pass in the grpc address and authentication credentials. These values can be passed in via the command line when running the `LNDK` program, like this:
+
+`cargo run -- <ADDRESS> <TLSPATH> <MACAROONPATH>`
```

Comment:
It's generally good practice as arguments such as `--something` which you want to pass to the binary and not cargo, will not work properly. In this case, since we just pass bare arguments, they do work just fine :)

Reply:
TIL!